### PR TITLE
Feature/style improvements

### PR DESCRIPTION
**Pull request by dfosco**
MAKING IT PRETTY :zap: :zap: :zap: :zap: :zap: :zap: :zap: :zap:

@amoreira @tamorim Please review and merge

### CODE REVIEW

**Comment by dfosco** [comment]
@amoreira @gkozlowaski pls review

**Comment by dfosco** [comment]
@gkozlowaski will do some adjustments to `Minicart.js` and `MinicartButton.js` to make markup more sensible before merging.

**Comment by dfosco** [comment]
@gkozlowaski will do some adjustments to make markup more sensible before merging. Rendering of `MinicartList` should be handled by `Minicart`, not `MinicartButton`.

**Comment by dfosco** [comment]
@amoreira please review

File: src/components/Minicart/MinicartButton/MinicartList/MinicartList.js [hunk 1]
```diff
@@ -34,18 +36,16 @@ class MinicartList extends React.Component {
...
+            <div className="MinicartList__total">
+              <span className="MinicartList__total-text">Total:</span>
+              <span className="MinicartList__total-price"><Price value={totalValue} /></span>
             </div>
-            <div className='col-lg-6 col-md-6'>
-              <a href="/checkout"><span className='btn btn-success'>Comprar</span></a>
+            <div className="MinicartList__cart-button">
+              <a href="/checkout"><button className="btn btn-success">Fechar Pedido</button></a>
             </div>
           </div>
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartList.js]: @dfosco Wrong indentation on the div

File: src/components/Minicart/MinicartButton/MinicartList/MinicartList.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import MinicartItem from './MinicartItem/MinicartItem'
 import './MinicartList.less'
 import { stores } from 'sdk'
 
+
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartList.js]: @dfosco Unnecessary newline

File: src/components/Minicart/MinicartButton/MinicartList/MinicartItem/MinicartItem.js [hunk 3]
```diff
@@ -14,33 +17,23 @@ class MinicartItem extends React.Component {
...
-            <div className='col-lg-6 col-md-6'>
-              <span>
-                Quantidade: {this.props.item.quantity}
-              </span>
-            </div>
-            <div className='col-lg-6 col-md-6'>
-              <span> <Price value={this.props.item.price/100} /> </span>
-            </div>
+      <li className="MinicartItem">
+      { this.props.isRemoveVisible ?
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartItem/MinicartItem.js]: @dfosco Please use a newline on block definitions, like:
```javascript
{
    this.props.isRemoveVisible ?
    ...
}
```

And indent accordingly.

File: src/components/Minicart/MinicartButton/MinicartButton.js [hunk 4]
```diff
@@ -22,36 +23,34 @@ class MinicartButton extends React.Component {
...
             <MinicartList items={this.props.items}
-                          itemsQuantity={this.props.itemsQuantity}
-                          removeFromCart={this.props.removeFromCart}
-                          updateCart={this.props.updateCart}
-                          isRemoveVisible={this.props.isRemoveVisible}
+            itemsQuantity={this.props.itemsQuantity}
+            removeFromCart={this.props.removeFromCart}
+            updateCart={this.props.updateCart}
+            isRemoveVisible={this.props.isRemoveVisible}
             /> :
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartButton.js]: @dfosco Try to be consistency on prop spacing.

File: src/components/Minicart/Minicart.less [hunk 5]
```diff
@@ -4,7 +4,7 @@
   .css-utils-clean-btn();
   .css-utils-no-outlines();
   width: 100%;
-  padding: 15px 0px;
+  padding: 15px 0px;
```
Comment (tamorim) [inline src/components/Minicart/Minicart.less]: @dfosco Trailing space.

File: src/components/Minicart/MinicartButton/MinicartList/MinicartItem/MinicartItem.js [hunk 6]
```diff
@@ -14,33 +17,23 @@ class MinicartItem extends React.Component {
   }
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartItem/MinicartItem.js]: @dfosco Newline required here

File: src/components/Minicart/MinicartButton/MinicartList/MinicartList.js [hunk 7]
```diff
@@ -34,18 +36,16 @@ class MinicartList extends React.Component {
                 />
               )
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartList.js]: @dfosco Missing semicolon

**Comment by tamorim** [comment]
@dfosco Also, there's a WIP commit on the history.

File: src/components/Minicart/MinicartButton/MinicartList/MinicartItem/MinicartItem.js (hunk 3 above)
Comment (dfosco) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartItem/MinicartItem.js]: @tamorim Is this the right identation?

```
      {
        this.props.isRemoveVisible ?
          <div className="MinicartItem__icon-wrapper" onClick={this.removeFromCart}>
            <SVGIcon svg={closeIcon} fallback={closeImg} className="MinicartItem__remove-icon" />
          </div>
        : null
      }
```

File: src/components/Minicart/MinicartButton/MinicartList/MinicartList.js [hunk 8]
```diff
@@ -12,16 +12,17 @@ class MinicartList extends React.Component {
 
     if (this.props.itemsQuantity === 0) {
       return(
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartList.js]: @dfosco Space after return
`return (`

File: src/components/Minicart/MinicartButton/MinicartList/MinicartList.js [hunk 9]
```diff
@@ -12,16 +12,17 @@ class MinicartList extends React.Component {
...
-          <span className='MinicartList__empty__text'>Carrinho vazio!</span>
+        <div>
+          <div className="MinicartList MinicartList__empty">
+          <span className="MinicartList__empty-text">Carrinho vazio!</span>
+          </div>
         </div>
       )
     }
 
     return(
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartList.js]: @dfosco Space after return
`return (`

File: src/components/Minicart/MinicartButton/MinicartList/MinicartList.js [hunk 10]
```diff
@@ -12,16 +12,17 @@ class MinicartList extends React.Component {
...
     if (this.props.itemsQuantity === 0) {
       return(
-        <div className='MinicartList MinicartList__empty'>
-          <span className='MinicartList__empty__text'>Carrinho vazio!</span>
+        <div>
+          <div className="MinicartList MinicartList__empty">
+          <span className="MinicartList__empty-text">Carrinho vazio!</span>
+          </div>
         </div>
       )
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartList.js]: @dfosco Semicolon at the end `);`

File: src/components/Minicart/MinicartButton/MinicartList/MinicartList.js [hunk 11]
```diff
@@ -32,20 +33,20 @@ class MinicartList extends React.Component {
...
-              )
+              );
             })}
-          </div>
         </ul>
-        <div className='container-fluid'>
-          <div className='row MinicartList__footer'>
-            <div className='col-lg-6 col-md-6'>
-              <span> Total: <Price value={totalValue} /> </span>
+          <div className="MinicartList__footer clearfix">
```
Comment (tamorim) [inline src/components/Minicart/MinicartButton/MinicartList/MinicartList.js]: @dfosco This div isnt well indented

**Comment by dfosco** [comment]
@tamorim Done! :zap: :zap: :zap:

**Comment by dfosco** [comment]
@tamorim Have you published a new version of the app? Still doesn't appear on the theme on master

**Comment by tamorim** [comment]
@dfosco Yup, I've published. It's a problem on the API that it's not auto promoting

**Pull request merged**